pkg/apis/kops/instancegroup.go

```diff
@@ -109,6 +109,16 @@ type InstanceGroupSpec struct {
 	Kubelet *KubeletConfigSpec `json:"kubelet,omitempty"`
 	// Taints indicates the kubernetes taints for nodes in this group
 	Taints []string `json:"taints,omitempty"`
+	// IAMProfileSpec defines the identity of the cloud group iam profile.
+	IAM *IAMProfileSpec `json:"iam,omitempty"`
```

Comment:
How about `InstanceGroupIAMSpec` ?  It isn't particularly visible to users, though it is visible to code clients etc.